sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorClient.cs

```diff
@@ -1223,7 +1232,17 @@ public virtual async Task<Response<IReadOnlyList<MetricEnrichedSeriesData>>> Get
         }
 
         /// <summary>
+        /// Gets data about the data points present in the specified set of time series.
+        /// <see cref="MetricAnomalyAlertConfiguration"/>.
         /// </summary>
+        /// <param name="seriesKeys">The name of the dimension.</param>
+        /// <param name="detectionConfigurationId">The unique identifier of the <see cref="MetricAnomalyAlertConfiguration"/>.</param>
+        /// <param name="startTime">Filters the result. Only data points ingested after this point in time, in UTC, will be returned.</param>
+        /// <param name="endTime">Filters the result. Only data points ingested after this point in time, in UTC, will be returned.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
+        /// <returns>A collection containing the values the specified dimension assumed for anomalous data points. Items are unique.</returns>
+        /// <exception cref="ArgumentNullException"><paramref name="seriesKeys"/> or <paramref name="detectionConfigurationId"/> is null.</exception>
+        /// <exception cref="ArgumentException"><paramref name="seriesKeys"/> or <paramref name="detectionConfigurationId"/> is empty; or <paramref name="detectionConfigurationId"/> is not a valid GUID.</exception>
```

Comment:
This description is very inaccurate. I think you copied it by accident from one the methods marked with TODODOCS (probably GetMetricEnrichedSeriesDataAsync).

Reply:
Not sure how I missed this, thanks.